### ISSUE TITLE
missing parentheses of a lambda call 

### ISSUE THREAD

**Issue by hzqmwne** [rocky/python-uncompyle6#350]
## Description

missing parentheses of a lambda call

## How to Reproduce

This is the source code:
```
A = (lambda x : x)(print)
```

then compile it with:
```
python3 -m compileall -b test.py
```
and decompile it
```
uncompyle6 test.pyc >test_dec.py
```
the output is:
```
A = lambda x: x(print)
```

the decompiled code is obviously error, that the parentheses are missing

## Expected behavior

expected decompiled code is:
```
A = (lambda x : x)(print)
```

## Environment

this bug can be repeated in such environments:
```
Ubuntu 20.04
Python 3.8.5 (default, May 27 2021, 13:30:53) [GCC 9.3.0] on linux
uncompyle6 3.7.4 from pip
```
or
```
Ubuntu 18.04
Python 3.6.9 (default, Jan 26 2021, 15:33:00) [GCC 8.4.0]
uncompyle6 3.7.4 from pip
```

## Additional Environment or Context

This bug also exists in [decompyle3](https://github.com/rocky/python-decompile3) 3.3.2

**Comment by rocky** [comment]
Fixed now in decompyle3 in commit  8c7e583c

I leave as an exercise for reader adapting this fix for this repository.  Note though that in this code base we don't generally use `x if condition else y` since we do try to keep compatible with older Pythons. 

The bug goes back all the way to Python 2.2 if not before (which is why I didn't ask about the version number earlier).

**Comment by rocky** [comment]
Fixed by #352
